site/src/pages/UsersPage/storybookData/users.ts

```diff
@@ -0,0 +1,577 @@
+import type { LoginType, User, UserStatus } from "api/typesGenerated";
```

Comment:
Should we replace all the emails with fake ones?  Some of these are probably real (the Coder ones at least, maybe the ones from ChatGPT too).

Reply:
Ah, sorry, I thought I had removed all of them. 🤦